Assignment/solution_student/css/main.css

```diff
@@ -0,0 +1,89 @@
+body {
+	border: 5px solid black;
+	border-top-style: solid;
+    border-right-style: none;
+    border-bottom-style: solid;
+    border-left-style: none;
+	width: 750px; 
+    padding-right: 30px;
+    padding-left: 30px;
+    margin: 20px auto;
+}
+
+body, nav {
```

Comment:
Because `font-family` is inherited, you don't actually need to declare it on the `nav` element, it will be inherited already, so you can move this rule into the main `body` declaration :)